web-bluetooth/scan.html

```diff
@@ -0,0 +1,39 @@
+---
+feature_name: Web Bluetooth / Scanning
+chrome_version: 79
```

Comment:
Let's make sure to update this if feature gets postponed.

Reply:
OK, also added note that chrome://flags/#enable-experimental-web-platform-features is required now.